sql/core/src/main/scala/org/apache/spark/sql/expressions/scalalang/typed.scala

```diff
@@ -77,6 +77,34 @@ object typed {
    */
   def sumLong[IN](f: IN => Long): TypedColumn[IN, Long] = new TypedSumLong[IN](f).toColumn
 
+  /**
+    * Min aggregate function for floating point (double) type.
+    *
+    * @since 2.3.0
+    */
```

Comment:
https://github.com/databricks/scala-style-guide#documentation-style

In Spark, we don't use the ScalaDoc style. Please correct all the description by using the multi-line JavaDoc comment style.